draft-ietf-capport-architecture.xml

```diff
@@ -601,10 +601,18 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
      <section title="Authenticated APIs">
        <t>
          The solution described here assumes that when the User Equipment needs to
-         trust the API server, server authentication will be utilized using TLS
+         trust the API server, server authentication will be performed using TLS
          mechanisms.
        </t>
      </section>
+     <section title="Secure APIs">
+       <t>
+           The solution described here requires that the API be secured using TLS.
+           This requirement follows from the fact that the user equipment may provide
+           secrets to the API. The API must ensure the integrity of this information,
```

Comment:
The current api includes secrets from the api, not the other way around.

Reply:
Thanks. I'll rephrase it to be more generic.

Reply:
Yes, perhaps something like:

"This is required to allow the user equipment and API server to exchange secrets which can be used to validate future interactions."

Reply:
Thanks, Tommy. I'll use that wording. :)